### PR TITLE
[WIP] Code refactoring using StyleCop.

### PR DESCRIPTION
**Pull request by kegor**
Replicated changes from https://github.com/groupdocs-viewer/GroupDocs.Viewer-for-.NET-WebForms/pull/70.

### CODE REVIEW

File: src/Products/Common/Resources/Resources.cs [hunk 1]
```diff
@@ -42,41 +43,45 @@ public static string GetFreeFileName(string directory, string fileName)
...
+        /// Generate exception for password error.
         /// </summary>
-        /// <param name="ex">Exception</param>
-        /// <param name="password">string</param>
-        /// <returns>ExceptionEntity</returns>
-        public ExceptionEntity GenerateException(System.Exception ex, String password)
+        /// <param name="ex">Exception.</param>
+        /// <param name="password">string.</param>
+        /// <returns>ExceptionEntity.</returns>
+        public ExceptionEntity GenerateException(System.Exception ex, string password)
```
Comment (bobkovalex) [inline src/Products/Common/Resources/Resources.cs:72]: Codacy found an issue: [Make 'GenerateException' a static method.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5915977)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
         

Clones removed
==============
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -3
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5915977&bid=19382972)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity increasing per file
==============================
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
         

Clones removed
==============
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5915976&bid=19382971)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 1
           

Complexity increasing per file
==============================
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
         

Clones removed
==============
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -5
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5915977&bid=19382972)

File: src/Products/Common/Resources/Resources.cs [hunk 2]
```diff
@@ -42,41 +42,45 @@ public static string GetFreeFileName(string directory, string fileName)
...
+        /// Generate exception for password error.
         /// </summary>
-        /// <param name="ex">Exception</param>
-        /// <param name="password">string</param>
-        /// <returns>ExceptionEntity</returns>
-        public ExceptionEntity GenerateException(System.Exception ex, String password)
+        /// <param name="ex">Exception object.</param>
+        /// <param name="password">Document password.</param>
+        /// <returns>Exception entity DTO-object.</returns>
+        public ExceptionEntity GenerateException(System.Exception ex, string password)
```
Comment (bobkovalex) [inline src/Products/Common/Resources/Resources.cs:71]: Codacy found an issue: [Make 'GenerateException' a static method.](https://app.codacy.com/gh/groupdocs-total/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5915977)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity increasing per file
==============================
- src/Products/Viewer/Controllers/ViewerApiController.cs  3
- src/Products/Viewer/Cache/MemoryPageStreamFactory.cs  1
         

Clones removed
==============
+ src/Products/Viewer/Controllers/ViewerApiController.cs  -5
+ src/Products/Viewer/Config/ViewerConfiguration.cs  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Total-for-NET-WebForms/pullRequest?prid=5915976&bid=19382971)

**Pull request merged**